packages/gafl-webapp-service/src/handlers/__tests__/agreed-handler-recurring-payments.spec.js

```diff
@@ -32,12 +38,13 @@ describe('The agreed handler', () => {
   })
   beforeEach(jest.clearAllMocks)
 
+  const mockTransactionCacheSet = jest.fn()
   const getMockRequest = (overrides = {}) => ({
     cache: () => ({
       helpers: {
         transaction: {
           get: async () => ({ cost: 0 }),
-          set: async () => {}
+          set: mockTransactionCacheSet
```

Comment:
This shouldn't be referring to an externally defined variable, please  define a new parameter. For the avoidance of confusion, it might be best to have an object config for the mock, e.g.
```
const getMockRequest = ({
  overrides = {},
  mockTransactionCacheSet = () => {}
} = {}) => ({
```
(I put mockTransactionCacheSet as a plain arrow function as the functionality of jest.fn isn't required most of the time)